backend/isic/crawlinginmyskin.py

```diff
@@ -0,0 +1,137 @@
+import requests
+import pandas as pd
+import pathlib
+import os
+
+"""A module to fetch data from International Skin Imaging Collaboration's API
+For further information about the API: https://isic-archive.com/api/v1/ """
+
+
+def ISIC_request(response, num=100):
+    """Fetching the metadata from images in ISIC archive
+    by sending a HTTP request to the ISIC's REST API
+
+    Parameters:
+
+    num(int):Number of the images you get from the archive,default is 100
+
+    Returns:
+
+    Json:Response metadata in JSON format"""
+    start_url = 'https://isic-archive.com/api/v1/image?limit=' + \
+        str(num)+'&sort=name&sortdir=-1&detail=true'
```

Comment:
Please use f-string instead.

```suggestion
    start_url = f'https://isic-archive.com/api/v1/image?limit={num}&sort=name&sortdir=-1&detail=true'
```